Alignment/OfflineValidation/python/TkAlAllInOneTool/configTemplates.py

```diff
@@ -124,6 +125,97 @@
 
 ######################################################################
 ######################################################################
+cfgTemplate="""
+import FWCore.ParameterSet.Config as cms
+
+process = cms.Process(".oO[ProcessName]Oo.")
+
+.oO[datasetDefinition]Oo.
+.oO[Bookkeeping]Oo.
+.oO[LoadBasicModules]Oo.
+.oO[TrackSelectionRefitting]Oo.
+.oO[LoadGlobalTagTemplate]Oo.
+.oO[condLoad]Oo.
+.oO[ValidationConfig]Oo.
+.oO[FileOutputTemplate]Oo.
+
+.oO[DefinePath]Oo.
+"""
+
+
+######################################################################
+######################################################################
+Bookkeeping = """
+process.options = cms.untracked.PSet(
+   wantSummary = cms.untracked.bool(False),
+   Rethrow = cms.untracked.vstring("ProductNotFound"), # make this exception fatal
+   fileMode  =  cms.untracked.string('NOMERGE') # no ordering needed, but calls endRun/beginRun etc. at file boundaries
+)
+
+process.load("FWCore.MessageLogger.MessageLogger_cfi")
+process.MessageLogger.cerr = cms.untracked.PSet(placeholder = cms.untracked.bool(True))
+process.MessageLogger.cout = cms.untracked.PSet(INFO = cms.untracked.PSet(
+reportEvery = cms.untracked.int32(1000) # every 1000th only
```

Comment:
Hi @hroskes thanks, if we agree that's fine for all validations, maybe warning level is enough? I am also open to other suggestions, like configuring it separately just for the PV.